packages/styletron-engine-atomic/src/cache.js

```diff
@@ -16,18 +18,28 @@ export class MultiCache<T> {
     this.idGenerator = idGenerator;
     this.onNewCache = onNewCache;
     this.onNewValue = onNewValue;
+    this.sortedCacheKeys = [];
     this.caches = {};
   }
 
   getCache(key: string): Cache<T> {
     if (!this.caches[key]) {
       const cache = new Cache(this.idGenerator, this.onNewValue);
       cache.key = key;
+      this.sortedCacheKeys.push(key);
+      this.sortedCacheKeys.sort(sortMq);
+      const insertAtIndex = this.sortedCacheKeys.findIndex(
```

Comment:
```suggestion
      const insertAtIndex = this.sortedCacheKeys.indexOf(key);
```

Reply:
I think `indexOf` might be better since it won't require any Array polyfills in older browsers. IIRC this codebase works with only prepublish transpilation.